samples/net/dsa/src/main.c

```diff
@@ -28,9 +33,9 @@ static void iface_cb(struct net_if *iface, void *user_data)
 				struct net_if *slave = dsa_get_slave_port(iface, i);
 
 				if (slave == NULL) {
-					LOG_ERR("Slave interface %d not found.", i);
-					break;
+					continue;
```

Comment:
Why we here allow the situation when slave can be NULL?

Reply:
It's possible some of switch ports are disabled in dts, for example, board design didn't export all ports.
For the disabled ports, the slave is NULL. We keep the index same with switch hardware in dsa driver level.
Thanks.